include/z64actor.h

```diff
@@ -493,18 +494,21 @@ typedef enum DoorLockType {
 // Player will retain this flag until the player is finished talking
 // Actor will retain this flag until `Actor_TalkOfferAccepted` is called or manually turned off by the actor
 #define ACTOR_FLAG_TALK (1 << 8)
-// 
-#define ACTOR_FLAG_200           (1 << 9)
-// 
-#define ACTOR_FLAG_400           (1 << 10)
-// 
-#define ACTOR_FLAG_800           (1 << 11)
+
+// When the hookshot attaches to this actor, the actor will be pulled back as the hookshot retracts.
+#define ACTOR_FLAG_HOOKSHOT_PULLS_ACTOR (1 << 9)
+
+// When the hookshot attaches to this actor, Player will be pulled by the hookshot and fly to the actor.
+#define ACTOR_FLAG_HOOKSHOT_PULLS_PLAYER (1 << 10)
+
+// Unused
+#define ACTOR_FLAG_800 (1 << 11)
```

Comment:
Just want to note that this is most likely a carry-over flag `ACTOR_FLAG_GRASS_DESTROYED` from OoT which is no longer used, but I decided not to comment since it's only speculative. Could write a comment though if others are interested

Reply:
Think its fair to say something to the effect of 
`// This is likely `ACTOR_FLAG_GRASS_DESTROYED` from OoT, however this flag is unused in this game.`